src/SCRIPTS/BF/PAGES/rescue.lua

```diff
@@ -5,8 +5,8 @@ return {
    title             = "GPS Rescue",
    reboot            = false,
    eepromWrite       = true,
-   minBytes          = 16,
-   requiredVersion   = 1.041,
+   minBytes          = 21,
+   requiredVersion   = 1.043,
```

Comment:
Seeing that we had a lot of improvements in the lua scripts since the last release, and will probably want to do another release before Betaflight 4.2 (which will bring API version 1.43) is released, I'd rather leave this at 1.041 for now - obviously the 'arming without fix' setting won't work for users before Betaflight 4.2, but at least they will still be able to use the rest of the GPS rescue page.

Reply:
but increased minBytes won't cause problems if script will not receive 21 bytes ?